src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -82,7 +83,7 @@ static bool processSubscriptions
                                          trigs->attrL,
                                          mapSubId,
                                          trigs->reference,
-                                         trigs->format,
+                                         JSON,
```

Comment:
I agree in not making this PR more complex. However, I think that for the sake of "cleaness" we should avoid having NotifyFormat and Format in the code base, unifying them under NotifyFormat. That means that part of the NGSI9 notification code needs a little bit of refactor to use NotifyFormat with hardwired NGSI_V1_JSON, but I think is not too much effort and worth the time (not for this PR and maybe not for the very next PR, but maybe for the next one after the very next PR :).